Dockerfile

```diff
@@ -32,7 +32,11 @@ RUN pnpm run build
 # The second stage builds the backend.
 FROM python:3.11-bullseye as backend-builder
 WORKDIR /phoenix
-COPY ./ /phoenix/
+COPY ./src /phoenix/src
+COPY ./pyproject.toml /phoenix/
+COPY ./LICENSE /phoenix/
+COPY ./IP_NOTICE /phoenix/
+COPY ./README.md /phoenix/
```

Comment:
Doesn't make sense to copy these here since they are not included in the final image.

Reply:
these are needed by the pip install